tables/sk-g1.ctb

```diff
@@ -1,6 +1,15 @@
+# liblouis: Slovak literary (grade 1) text braille translation table
 #
-#  Copyright (C) 2014 by Branislav Mamojka <mamojka@unss.sk>
-#  Copyright (C) 2016 by Peter Vagner <peter.v@datagate.sk>
+#-index-name: Slovak
+#-display-name: Slovak literary braille
```

Comment:
The generated display name will be just "Slovak braille", without "literary", because Liblouis currently does not have other Slovak tables. Is there a standard for computer braille?

Reply:
The standard for computer braille or other grade is not yet formalized and official spec is not published. It's known some people are using both computer braille and something that may resemble grade 2 in the wild but none of that is standardized yet.

Reply:
Oh, I just realized we never include "literary" in the display name (see [extra/generate-display-names/display-names](https://github.com/liblouis/liblouis/blob/master/extra/generate-display-names/display-names) for the overview). So please remove it. By the way, if you want there is also the `#-name` metadata field which you can use for a name/description of the braille code in the native language (Slovak).

Reply:
Fixed in a7c2a462c5d70ed75b99a3d7fd658f499147d3b1